File: templates/_composition/MVVMLight/Page.MasterDetail.AddViewModel/ViewModels/wts.ItemNameDetailViewModel_postaction.cs
//{[{
using GalaSoft.MvvmLight;
using GalaSoft.MvvmLight.Command;
//}]}
namespace Param_ItemNamespace.ViewModels
{
    public class wts.ItemNameDetailViewModel : ViewModelBase
    {
        //{[{
        public NavigationServiceEx NavigationService
        {
            get
            {
                return Microsoft.Practices.ServiceLocation.ServiceLocator.Current.GetInstance<NavigationServiceEx>();
            }
        }

        //}]}
    }
}

File: templates/_composition/MVVMLight/Page.MasterDetail.AddViewModelVB/ViewModels/wts.ItemNameDetailViewModel_postaction.vb
'{[{
Imports GalaSoft.MvvmLight
Imports GalaSoft.MvvmLight.Command
'}]}
Namespace ViewModels
    Public Class wts.ItemNameDetailViewModel
        Inherits ViewModelBase
        '{[{
        Public ReadOnly Property NavigationService() As NavigationServiceEx
            Get
                Return Microsoft.Practices.ServiceLocation.ServiceLocator.Current.GetInstance(Of NavigationServiceEx)()
            End Get
        End Property

        '}]}
    End Class
End Namespace
